apps/hash-frontend/src/pages/settings/organizations/[shortname]/members.page.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  styled,
+  Table,
+  TableBody,
+  TableCell,
+  TableFooter,
+  TableHead,
+  TableRow,
+  Typography,
+} from "@mui/material";
+import { useRouter } from "next/router";
+import { NextSeo } from "next-seo";
+import { useRef, useState } from "react";
+
+import { NextPageWithLayout } from "../../../../shared/layout";
+import { useAuthenticatedUser } from "../../../shared/auth-info-context";
+import { getSettingsLayout } from "../../shared/settings-layout";
+import { Cell } from "../shared/cell";
+import { OrgSettingsContainer } from "../shared/org-settings-container";
+import { AddMemberForm } from "./members.page/add-member-form";
+import { MemberRow } from "./members.page/member-row";
+
+const InviteNewButton = styled("button")`
+  background: none;
+  border: none;
+  cursor: pointer;
+  font-weight: 500;
+  padding: 0;
+`;
+
+const OrgMembersPage: NextPageWithLayout = () => {
+  const router = useRouter();
+
+  const topRef = useRef<HTMLSpanElement>(null);
+
+  const { shortname } = router.query as { shortname: string };
+
+  const { authenticatedUser } = useAuthenticatedUser();
+
+  const [showAddMemberForm, setShowAddMemberForm] = useState(false);
+
+  const org = authenticatedUser.memberOf.find(
+    (orgOption) => orgOption.shortname === shortname,
+  );
+
+  if (!org) {
+    // @todo show a 404 page
+    void router.push("/");
+    return null;
+  }
+
+  return (
+    <>
+      <NextSeo title={`${org.name} | Members`} />
+
+      <OrgSettingsContainer
+        header={org.name}
+        sectionLabel="Members"
+        ref={topRef}
+      >
```

Comment:
This and the layout function at the bottom are the only changes in this and other existing pages that appear deleted but have been moved